Shared/Samples/Display content of utility network container/DisplayContentOfUtilityNetworkContainerView.Model.swift

```diff
@@ -0,0 +1,278 @@
+            // Gets the containment associations from the element to display its content.
+            let containmentAssociations = try await network.associations(for: containerElement, ofKind: .containment)
+            // Determines the relationship of each element and add it to the content elements.
+            return containmentAssociations.map { association in
+                if association.fromElement.objectID == containerElement.objectID {
+                    return association.toElement
+                } else {
+                    return association.fromElement
+                }
+            }
+        }
+        
+        /// Creates the graphics for the utility elements within a container.
+        /// - Parameters:
+        ///   - contentElements: The elements within the container.
+        ///   - containerElement: The container element.
+        /// - Returns: An array of graphics.
+        private func makeGraphics(for contentElements: [UtilityElement], within containerElement: UtilityElement)  async throws -> [Graphic] {
+            let contentFeatures = try await network.features(for: contentElements)
+            let graphics: [Graphic] = contentFeatures.compactMap { feature in
+                guard let featureTable = feature.table as? ServiceFeatureTable,
+                      let symbol = featureTable.layerInfo?.drawingInfo?.renderer?.symbol(for: feature) else {
+                    return nil
+                }
+                return Graphic(geometry: feature.geometry, symbol: symbol)
+            }
+            return graphics
+        }
+        
+        /// Creates the associations graphics for elements in the container.
+        /// - Parameter boundingExtent: The boundary for the container.
+        /// - Returns: An array of association relationship graphics.
+        private func makeGraphics(forAssociationsWithin boundingExtent: Envelope) async throws -> [Graphic] {
+            let containmentAssociations = try await network.associations(forExtent: boundingExtent)
+            let graphics: [Graphic] = containmentAssociations.map { association in
+                let symbol = association.kind == .attachment ? attachmentSymbol : connectivitySymbol
+                return Graphic(geometry: association.geometry, symbol: symbol)
+            }
+            return graphics
+        }
+        
+        // MARK: Helpers
+        
+        /// Changes the visibility of the operational layers.
+        /// - Parameter isVisible: A Boolean to make the map visible or not.
+        func setOperationalLayersVisibility(isVisible: Bool) {
+            map.operationalLayers.forEach { $0.isVisible = isVisible }
+        }
+        
+        /// Sets status message.
+        /// - Parameter message: A message of the current status.
+        private func setStatusMessage(_ message: String) {
+            statusMessage = message
+        }
```

Comment:
Please remove this method and instead set the `statusMessage` property directly.